### PR TITLE
added Dockerfile for agent-examples

### PR DESCRIPTION
**Pull request by dvagapov**

### CODE REVIEW

File: examples/README.md [hunk 1]
```diff
@@ -60,8 +60,18 @@ mvn test -P examples -D spline.mode=REQUIRED
   - [Python](src/main/python/)
   - [Shell script](src/main/shell/) - custom, non-Spark example, using REST API
 
+## Run Spline examples using docker image
 
- 
+1. Build docker image
+```shell script
+# from /root folder build docker image
```
Comment (wajda) [inline examples/README.md]:
```suggestion
# build a docker image from the project root folder
```

File: examples/README.md [hunk 2]
```diff
@@ -60,8 +60,18 @@ mvn test -P examples -D spline.mode=REQUIRED
...
+docker build -f examples/Dockerfile . -t spline-spark-agent-examples:v0.1.0
+```
+
+2. Run docker image
+```shell script
+docker run -e "SPLINE_PRODUCER_URL=http://localhost:8080/producer" spline-spark-agent-examples:v0.1.1
+```
```
Comment (wajda) [inline examples/README.md]: missing indentation (the same in the previous block)
```suggestion
    ```shell script
    docker run -e "SPLINE_PRODUCER_URL=http://localhost:8080/producer" spline-spark-agent-examples:v0.1.1
    ```
```

**Review by wajda — COMMENTED** [review]
Thanks Danil!

There are two minor comments, otherwise LGTM

File: examples/README.md (hunk 1 above)
Comment (dvagapov) [reply examples/README.md]: Changed

Comment (dvagapov) [reply examples/README.md]: changed

File: examples/README.md (hunk 2 above)
Comment (dvagapov) [reply examples/README.md]: changed

**Review by wajda — APPROVED** [review]
LGTM

**Pull request merged**